src/memalloc.jl

```diff
@@ -55,8 +55,8 @@ isfuncexpr(ex::Expr) =
     ex.head == :function || (ex.head == :(=) && typeof(ex.args[1]) == Expr && ex.args[1].head == :call)
 isfuncexpr(arg) = false
 
-# Support Unix command line usage like `julia Coverage.jl $(find ~/.julia/v0.3 -name "*.jl.mem")`
-if !isinteractive()
+# Support Unix command line usage like `julia Coverage.jl $(find ~/.julia/v0.6 -name "*.jl.mem")`
+if abspath(PROGRAM_FILE) == joinpath(dirname(@__FILE__), "Coverage.jl")
```

Comment:
You can use `@__DIR__` instead of `dirname(@__FILE__)`. This should work on all versions of Julia since Compat is already requiring Compat 0.9.5.